financialaid/templates/review_financial_aid.html

```diff
@@ -0,0 +1,112 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block title %}{% trans "Review Financial Aid" %}{% endblock %}
+
+{% block content %}
+  {% include "header.html" %}
+  <div class="page-content">
+    <div class="single-column" style="max-width: 1000px;">
+      <p><h4>Financial Aid Submissions</h4></p>
+      <div class="mdl-card">
+        <!-- Sorting buttons -->
+        <div>
+          <strong>Sort by: </strong>
+          <div class="btn-group">
+          {% for field in sort_field_info %}
+            <a href="{% url 'review_financial_aid' program_id=current_program_id status=selected_status %}?sort_by={{ field.sort_field }}"
+               class="btn btn-default btn-sm">
+              {{ field.display }}
+              {% if field.direction_display == "" %}&#9650;{% endif %}
+              {% if field.direction_display == "-" %}&#9660;{% endif %}
+            </a>
+          {% endfor %}
+          </div>
+        </div>
+        <!-- /Sorting buttons -->
+        <!-- Filters -->
+        <div>
+          <select class="form-control" onchange="location = this.options[this.selectedIndex].value;">
+          {% for status, message in financial_aid_statuses %}
+            <option value="{% url 'review_financial_aid' program_id=current_program_id status=status %}"
+                    {% if selected_status == status %}selected{% endif %}>
+              {{ message }}
+            </option>
+          {% endfor %}
+          </select>​
+        </div>
+        <!-- /Filters -->
+        <!-- Pagination buttons -->
+        <div>
+          <span class="step-links">
+          {% if page_obj.has_previous %}
+            <a href="?page={{ page_obj.previous_page_number }}" class="btn btn-default"><</a>
```

Comment:
That's an intentional < for the arrow on the button - I can change it to an html code instead?